src/plugins/user_setup/kibana.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "userSetup",
+  "version": "8.0.0",
+  "kibanaVersion": "kibana",
+  "configPath": ["userSetup"],
+  "server": true,
```

Comment:
Should this have owner and description properties?

Reply:
Ah, good point, forgot about these new fields. Will add them, thanks!

Reply:
Done in https://github.com/elastic/kibana/pull/101610/commits/86243968834d34d407ed583b16a5e52eaaf6790e